client/src/components/to-do-list/Tasks.jsx

```diff
@@ -3,15 +3,23 @@
 import React from 'react';
 import Task from './Task.jsx';
 
-function Tasks({tasks, editTask, deleteTask}) {
+function Tasks({tasks, openModal, editClick, editing, deleteTask}) {
+  // console.log('tasks in tasks', tasks)
   return(
     tasks.map((task, i) => {
       if (task.inCalendar === false) {
         console.log('TASKS task: ', task);
-        return (<Task key={i} index={i} title={task.title} task={task} deleteTask={deleteTask}/>);
+        return (<Task style={{display: 'inline-block'}} key={i} task={task} openModal={openModal} editClick={editClick} editing={editing} deleteTask={deleteTask}/>)      
```

Comment:
Are there going to be different display styles for a Task? If not probably best to have this inside the Task component